modules/tinymce/src/plugins/help/main/ts/ui/KeyboardNavTab.ts

```diff
@@ -3,68 +3,91 @@ import { Dialog } from 'tinymce/core/api/ui/Ui';
+const description = `<h1>Activate keyboard navigation</h1>
 
-<h2>Activating keyboard navigation</h2>
+<dl>
+  <dt>Activate the Menu bar</dt>
+  <dd>Windows or Linux: Alt+F9</dd>
+  <dd>macOS: &#x2325;+F9</dd>
+  <dt>Activate the Toolbar</dt>
+  <dd>Windows or Linux: Alt+F10</dd>
+  <dd>macOS: &#x2325;+F10</dd>
+  <dt>Activate the Document footer</dt>
+  <dd>Windows or Linux: Alt+F11</dd>
+  <dd>macOS: &#x2325;+F11</dd>
+  <dt>Activate an open context menu or toolbar</dt>
+  <dd>Windows, Linux or macOS: Ctrl+F9
+</dl>
 
-<p>The sections of the outer UI of the editor - the menubar, toolbar, sidebar and footer - are all keyboard navigable. As such, there are multiple ways to activate keyboard navigation:</p>
-<ul>
-  <li>Focus the menubar: Alt + F9 (Windows) or &#x2325;F9 (MacOS)</li>
-  <li>Focus the toolbar: Alt + F10 (Windows) or &#x2325;F10 (MacOS)</li>
-  <li>Focus the footer: Alt + F11 (Windows) or &#x2325;F11 (MacOS)</li>
-</ul>
+<p>Navigation will start at the first UI item, which will be highlighted, or underlined in the case of the first item in the Footer element path.</p>
 
-<p>Focusing the menubar or toolbar will start keyboard navigation at the first item in the menubar or toolbar, which will be highlighted with a gray background. Focusing the footer will start keyboard navigation at the first item in the element path, which will be highlighted with an underline. </p>
 
-<h2>Moving between UI sections</h2>
+<h1>Navigate between UI sections</h1>
 
-<p>When keyboard navigation is active, pressing tab will move the focus to the next major section of the UI, where applicable. These sections are:</p>
-<ul>
-  <li>the menubar</li>
-  <li>each group of the toolbar </li>
-  <li>the sidebar</li>
-  <li>the element path in the footer </li>
-  <li>the wordcount toggle button in the footer </li>
-  <li>the branding link in the footer </li>
-  <li>the editor resize handle in the footer</li>
-</ul>
+<p>To move from one UI section to the next, press <strong>Tab</strong>.</p>
+
+<p>To move from one UI section to the previous, press <strong>Shift+Tab</strong>.</p>
+
+<p>The <strong>Tab</strong> order of these UI sections is:
+
+<ol>
+<li>Menu bar</li>
+<li>Toolbar group 1</li>
+<li>Toolbar group n</li>
+<li>Sidebar</li>
+<li>Document footer</li>
+<li>Word count toggle button in the TinyMCE footer</li>
+<li>Branding link in the TinyMCE footer</li>
+<li>Editor resize handle in the TinyMCE footer</li>
+</ol>
 
-<p>Pressing shift + tab will move backwards through the same sections, except when moving from the footer to the toolbar. Focusing the element path then pressing shift + tab will move focus to the first toolbar group, not the last.</p>
+<p>If a UI section is not present, it is skipped.</p>
 
-<h2>Moving within UI sections</h2>
+<p>If the document footer has keyboard navigation focus, and there is no visible sidebar, pressing <strong>Shift+Tab</strong> moves focus to <strong>Toolbar group 1</strong>, not <strong>Toolbar group n</strong>.
```

Comment:
Clearer to state the "first" and "last" rather than "1" and "n", especially for translation purposes.
```suggestion
<p>If the footer has keyboard navigation focus, and there is no visible sidebar, pressing <strong>Shift+Tab</strong> moves focus to the first toolbar group, not the last.
```